main.c

```diff
@@ -157,91 +182,439 @@ int main(void)
         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
     };
-    psa_key_policy_t policy = PSA_KEY_POLICY_INIT;
-    static uint8_t pubkey[pubkey_size];
-    size_t pubkey_len = 0;
-    psa_key_slot_number_t atecc608a_key_slot_device = 0;
-    psa_key_slot_number_t atecc608a_public_key_slot = 9;
 
-    atecc608a_print_serial_number();
-    atecc608a_print_config_zone();
+    ASSERT_SUCCESS_PSA(atecc608a_drv_info.p_asym->p_sign(
+                       atecc608a_private_key_slot, alg, hash, sizeof(hash),
```

Comment:
There are many cases of misleading indentation in this file. Please reindent the whole file in your favorite IDE or with `astyle --options=/path/to/mbedos/.astylerc`.

Reply:
Done with astyle and the mbed-os style.